storage.go

```diff
@@ -49,6 +50,19 @@ func (s *Storage) copy(ctx context.Context, src string, dst string, opt pairStor
 	return nil
 }
 
+func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
+	// FIXME: should set false or true here?
+	o = s.newObject(true)
+	o.ID = filepath.Join(s.workDir, path)
+	o.Path = path
+	o.Mode = ModeRead
```

Comment:
We need to update `Mode` if we have `multipart_id`, as for qingstor, multupart object is not readable.